pset-functions.js

```diff
@@ -207,6 +277,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 5:')
 
 // Add your code below this line
+function doubleLettersCheck(word){
+  const splitWords = word.split("");
+  for (let i = 0; i < splitWords.length; i++){
+    if(splitWords[i] === splitWords[i + 1]){
+    console.log("true")} else{ console.log("false");}
```

Comment:
@cbrooks2019 same comment as above, you should just return the value (in this case `true` or `false`) rather than printing value. You should print the value outside the function, which you are already doing. 

Take a look at the solution for an example of that approach in practice: https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L300

Notice, how the function uses the `return` keyword to return either `true` or `false`